types/ng-i18next/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/i18next/ng-i18next
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
 /// <reference types="i18next" />
```

Comment:
Here also TypeScript 2.3 header is not required, but you should remove `/// <reference ...`.